akka-stream-testkit/src/test/scala/akka/stream/testkit/ChainSetup.scala

```diff
@@ -22,8 +22,8 @@ class ChainSetup[In, Out](
            toPublisher: (Source[Out, _], ActorFlowMaterializer) ⇒ Publisher[Out])(implicit system: ActorSystem) =
```

Comment:
Also ChainSetup feels a bit old-school, before the times where Source and Flow has been separated. I guess this has been ported quite literally, but this could be expressed much easier with the new DSL. We should not expose this anyway to users.